cidc_api/models/files/facets.py

```diff
@@ -209,6 +209,7 @@ def __init__(self, match_clauses: List[str], description: Optional[str] = None):
         "Images": FacetConfig(["/ihc/ihc_image."]),
         "Combined Markers": FacetConfig(["csv|ihc marker combined"]),
     },
+    "H&E": {"Images": FacetConfig(["hande/image_file."], "Stained image files.")},
```

Comment:
Should this be `hande/image_file.svs`?
https://github.com/CIMAC-CIDC/cidc-schemas/blob/886273228354a2ac3cbbf8592798b48fd7566dd4/cidc_schemas/schemas/templates/metadata/hande_template.json#L29